SU-HSDO/suhumsci
# ARCH-175 reduced spacing between cards updated related mixin

# READY FOR REVIEW

# Summary
- Updated card mixin to reduce spacing between cards (reduced margin-bottom)

# Needed By (Date)
- End of sprint (7/19)

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Ensure that the margin-bottom of cards is 1.5em

# Affected Projects or Products
- Archeaology
- suhumsci (theme)

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/ARCH-175
- BUGHERD-29

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
